### ISSUE TITLE
Array and string offset access syntax with curly braces is deprecated

### ISSUE THREAD

**Issue by lukyrys** [nullpunkt/lz-string-php#22]
Array and string offset access syntax with curly braces is deprecated for php 7.4 at https://github.com/nullpunkt/lz-string-php/blob/master/src/LZCompressor/LZString.php#L59

**Comment by dducro** [comment]
Same issue.

```
PHP Deprecated:  Array and string offset access syntax with curly braces is deprecated in 
```
https://github.com/nullpunkt/lz-string-php/blob/master/src/LZCompressor/LZString.php#L22

**Comment by dducro** [comment]
PR #24 fixes this issue.

**Comment by nullpunkt** [comment]
Fixed by PR #24
